main/lsp/requests/hover.cc

```diff
@@ -21,14 +22,28 @@ string methodSignatureString(const core::GlobalState &gs, const core::TypePtr &r
         }
         start = start->secondary.get();
     }
+
+    contents = absl::StripTrailingAsciiWhitespace(contents);
```

Comment:
Is this needed? Does this method produce strings that have extra trailing whitespace?

Reply:
Hmm I think this was a follow-up from https://github.com/sorbet/sorbet/pull/1499#discussion_r311180435 but now that you mention it, it doesn't seem like it would. `methodDetail`'s returns are always without newline. I'll remove this line for now -- f5207200b